src/homework2/Arrays/Task4.java

```diff
@@ -0,0 +1,30 @@
+package homework2.Arrays;
+
+import java.util.Arrays;
+
+public class Task4 {
+    public static void main(String[] args) {
+        // ---------------------------------------------------------
+        // task - 4: Write a program that reads an array and print whether it is mirrored. The following arrays are mirrored:
+        // ---------------------------------------------------------
+
+        int[] arr = { 3 };
+
+        if (arr.length == 1) {
+            System.out.println("Array is mirrored.");
+        } else {
+            boolean isMirroredArray = false;
+
+            for (int i = 0; i < arr.length / 2; i++) {
```

Comment:
if you are checking both ends the loop should be till the salf of the length:
i < arrayHalf;